cf-java-logging-support-opentelemetry-agent-extension/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>cf-java-logging-support-opentelemetry-agent-extension</artifactId>
+    <packaging>jar</packaging>
+
+    <name>cf-java-logging-support-opentelemetry-agent-extension</name>
+
+    <parent>
+        <artifactId>cf-java-logging-support-parent</artifactId>
+        <groupId>com.sap.hcp.cf.logging</groupId>
+        <version>3.7.1</version>
+    </parent>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <opentelemetry.sdk.version>1.31.0</opentelemetry.sdk.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.opentelemetry</groupId>
+                <artifactId>opentelemetry-bom</artifactId>
+                <version>${opentelemetry.sdk.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>io.opentelemetry</groupId>
+                <artifactId>opentelemetry-bom-alpha</artifactId>
+                <version>${opentelemetry.sdk.version}-alpha</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-sdk-common</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.pivotal.cfenv</groupId>
+            <artifactId>java-cfenv</artifactId>
+            <version>2.5.0</version>
+        </dependency>
```

Comment:
Instead of using a CF-specific library to retrieve service bindings you might want to take a look at the [BTP Service Binding Access](https://github.com/SAP/btp-environment-variable-access) lib, that was developed and is used by SAP Cloud SDK, CAP Java and the SAP Security Lib. It allows to retrieve service binding information independent if you are running on CF or Kyma (or any other environment supporting the Open Service Broker API).

If that sounds interesting to you, maybe I can also try a contribution to refactor the existing code based on the library.